syslog/sandboxes/heka/input/syslog_unixsock.lua

```diff
@@ -0,0 +1,126 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This duplicates a lot of the syslog_udp code; the common code should be factored out or the plugins combined.